src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -1466,6 +1467,82 @@ namespace OpenLoco::Ui::WindowManager
         invalidateWidget(window->type, window->number, widgetIndex);
     }
 
+    static bool isStepperGroup(Window& w, WidgetIndex_t index, WidgetType buttonType)
+    {
+        const auto& widgets = w.widgets;
+
+        if (widgets[index].type != WidgetType::textbox && widgets[index].type != WidgetType::wt_3)
+            return false;
+
+        if (widgets[index + 1].type != buttonType)
+            return false;
+
+        if (widgets[index + 2].type != buttonType)
+            return false;
+
+        return true;
+    }
+
+    static std::optional<WidgetIndex_t> getStepperGroupWidgetIndex(Window& w, WidgetIndex_t startIndex)
+    {
+        // We only iterate 3 times as we might be at the the textbox or one of the buttons.
+        for (WidgetIndex_t index = 0; index < 3; index++)
```

Comment:
this `3` looks used in a few spots, can we add a constant for it?

Reply:
That's the only place its used and its quite specific to this function as the group consists of 3 widgets.